### PR TITLE
update code example

### PR DESCRIPTION
**Pull request by nohzafk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,10 +49,13 @@ Please run `tetos --help` for available providers and options.
...
 
 ```python
+import anyio
+
+from pathlib import Path
 from tetos.azure import AzureSpeaker
 
 speaker = AzureSpeaker(speech_key='...', speech_region='...')
-speaker.say('Hello, world!', 'output.mp3')
+anyio.run(speaker.synthesize, 'Hello, world!', Path('output.mp3'))
```
Comment (frostming) [inline README.md:58]: `speaker.say` is the synchronous version that wraps `synthesize` inside `anyio.run`, any reason to prefer that?

`Path('output.mp3')` is needed. Thanks

Comment (nohzafk) [reply README.md:58]: I just tried it few seconds ago and saw this, i though the function is not existed.

I use python3.11 and install tetso using `pip install tetso` 

```shell
Traceback (most recent call last):
  File "/Users/randall/projects/tts-tetos/tts2.py", line 8, in <module>
    speaker.say('Hello, world!', 'hello.mp3')
    ^^^^^^^^^^^
AttributeError: 'AzureSpeaker' object has no attribute 'say'
```

Comment (nohzafk) [reply README.md:58]: OK, I figured it out, the PYPI package is out of date. You may need to update it in order for the code example to work.

Comment (frostming) [reply README.md:58]: just released 0.0.2 it should work now

**Pull request closed without merging**